packages/snjs/lib/services/api/http_service.ts

```diff
@@ -30,6 +32,15 @@ export type HttpRequest = {
  * A non-SNJS specific wrapper for XMLHttpRequests
  */
 export class SNHttpService extends PureService {
+  constructor(
+    private readonly environment: Environment,
+    private readonly appVersion: string
+  ) {
+    super();
+
+    this.appVersion = appVersion;
```

Comment:
This can be removed now, because the TS shorthand notation automatically does this :)

Reply:
Ah, sure, forgot removing them, thanks! :)